### ISSUE TITLE
NTR: N-methylhistamine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8849]
New term request N-methylhistamine high in urine (elthat_111113035510):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000533
- label: Elevated urinary N-methylhistamine level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:12372095
- def: The amount of N-methylhistamine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8849
- synonyms:
